sdk/src/main/scala/coop/rchain/sdk/dag/merging/ConflictResolutionLogic.scala

```diff
@@ -7,7 +7,7 @@ import scala.collection.compat.immutable.LazyList
 import scala.collection.mutable
 import scala.math.Numeric.LongIsIntegral
 
-object DagMergingLogic {
+object ConflictResolutionLogic {
```

Comment:
Since this is no longer have anything with the DAG, it's just a conflict resolution logic.